src/SDKs/Compute/Management.Compute/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Compute Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Compute Resources.")]
 
-[assembly: AssemblyVersion("15.0.0.0")]
-[assembly: AssemblyFileVersion("15.0.0.0")]
+[assembly: AssemblyVersion("16.0.0.0")]
```

Comment:
@hyonholee just curious, why AssemblyVersion and nuget version differed earlier (nuget version being 15, but AssemblyVersion 16)
And now why update from 16.0 to 16.2? why skip 16.1?